arrays_strings/unique_chars/unique_chars_solution.ipynb

```diff
@@ -2,21 +2,30 @@
  "cells": [
   {
    "cell_type": "markdown",
-   "metadata": {},
+   "metadata": {
```

Comment:
Do we know what are these new metadata tags?  They are making the diff a little more difficult to review.  What version of Jupyter are you using, perhaps this is in the latest version?